.github/workflows/python-build-test.yaml

```diff
@@ -94,13 +94,17 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: |
+            3.8
+            3.9
+            3.10
+            3.11
 
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
           target: ${{ matrix.target }}
-          args: --release --out dist --find-interpreter --manifest-path pylace/Cargo.toml
```

Comment:
Does find-interpreter not work in this case?

Reply:
`--find-interpreter` SHOULD work but I thought it would be better to be explicit about what python versions we wanted; that way we would know if we weren't building the versions we want.

I'm not sure what happens if you combine `---find-interpreter` with `--interpreter`

Reply:
```
❯ maturin build --release --out dist  --manifest-path pylace/Cargo.toml -i python3.10  -i python3.11 -i python3.13 --find-interpreter
error: The argument '--interpreter [<INTERPRETER>...]' cannot be used with '--find-interpreter'

Usage: maturin build --release --out <OUT> --manifest-path <PATH> --interpreter [<INTERPRETER>...] [ARGS]...

For more information try '--help'
```

so that's that